Shared/Samples/Manage operational layers/ManageOperationalLayersView.swift

```diff
@@ -26,128 +26,122 @@ struct ManageOperationalLayersView: View {
                     Button("Manage Layers") {
-                        isShowingSheet = true
+                        layersManagerIsPresented = true
                     }
-                    .popover(isPresented: $isShowingSheet) {
-                        ManageLayersSheetView(map: map)
-                            .presentationDetents([.fraction(0.5)])
-                            .frame(idealWidth: 320, idealHeight: 360)
+                    .popover(isPresented: $layersManagerIsPresented) {
+                        LayersManager(map: map, layers: operationalLayers)
+                            .presentationDetents([.fraction(0.4)])
+                            .frame(idealWidth: 320, idealHeight: 260)
                     }
                 }
             }
-            .errorAlert(presentingError: $error)
     }
 }
 
-struct ManageLayersSheetView: View {
-    /// The map with the operational layers.
+/// A view for managing the layers of a given map.
+private struct LayersManager: View {
+    /// The map to manage.
     let map: Map
     
-    /// The action to dismiss the manage layers sheet.
+    /// The layers to add to and remove from the map.
+    let layers: [Layer]
+    
+    /// The action to dismiss the view.
     @Environment(\.dismiss) private var dismiss
     
-    /// An array for all the layers currently on the map.
+    /// The layers currently added to the map.
     @State private var operationalLayers: [Layer] = []
     
-    /// An array for all the layers removed from the map.
+    /// The layers removed from the map.
     @State private var removedLayers: [Layer] = []
     
     var body: some View {
-        VStack {
-            ZStack {
-                Text("Manage Layers")
-                    .bold()
-                HStack {
-                    EditButton()
-                    Spacer()
-                    Button("Done") {
-                        dismiss()
-                    }
-                }
-            }
-            .padding([.top, .leading, .trailing])
-            
-            List {
+        NavigationStack {
+            Form {
                 Section {
                     ForEach(operationalLayers, id: \.id) { layer in
                         HStack {
-                            Image(systemName: "minus.circle.fill")
-                                .foregroundStyle(.red)
-                                .imageScale(.large)
-                                .clipped()
-                                .onTapGesture {
-                                    // Remove layer from map on minus press.
-                                    map.removeOperationalLayer(layer)
+                            Button("Remove Layer", systemImage: "minus.circle.fill") {
+                                map.removeOperationalLayer(layer)
+                                
+                                withAnimation {
+                                    operationalLayers.removeAll { $0.id == layer.id }
```

Comment:
FYI: The closure parameter is unmuted (has a label) for a reason. Apple could have designed it without a label (like they dd with `map(_:)`, but they didn't. I would suggest keeping the label.

```suggestion
                                    operationalLayers.removeAll(where: { $0.id == layer.id })
```